keps/prod-readiness/sig-node/375.yaml

```diff
@@ -0,0 +1,7 @@
+kep-number: 375
```

Comment:
this looks to be linked to https://github.com/kubernetes/enhancements/issues/3570 for kube enhancements tracking

Reply:
Done. I updated the KEP number with the newest one, because it's a bit more practical to work on. Added backlinks (and used `git mv`) to preserve the history and have clear history. I think this should be good enough for the project purposes; if this is not the case I'd be most happy to change direction.